channels/timed_set.go

```diff
@@ -68,13 +74,20 @@ func (vbs VbSequence) Equals(other VbSequence) bool {
 
 // A mutable mapping from channel names to sequence numbers (interpreted as the sequence when
 // the channel was added.)
-type TimedSet map[string]VbSequence
+type TimedSet map[string]TimedSetEntry
+
+type TimedSetEntry struct {
```

Comment:
Is there a reason to not put `Source` in `VbSequence` ? It seems like all sequences are going to have a `Source` assigned to them. I think doing this would make it so many fewer lines of code have to be touched. 

`VbSequence` name itself is somewhat misleading because I don't think `VbNo` is ever used anymore but I'd not change this as part of this PR.

I'd document that `Source` is not always set.

Reply:
Added comment